### ISSUE TITLE
Failed to Authenticate

### ISSUE THREAD

**Issue by Vibinreji** [mitchtabian/Android-Instagram-Clone#31]
Hi,When i try to register new user i am getting error toast like "Failed to Authenticate". So that i couldnt use the App.Please help me

**Comment by stefanionescu** [comment]
Nope, I get the same error even if I use my own Firebase account with my own google-services and I also changed the package name
